x-pack/plugins/fleet/common/types/models/agent.ts

```diff
@@ -47,6 +47,7 @@ export interface NewAgentAction {
   start_time?: string;
   minimum_execution_duration?: number;
   source_uri?: string;
+  total?: number;
```

Comment:
Added the `total` count that represents how many agents were actioned (clicked by user), this helps with status reporting in case something went wrong while creating the action documents in batches.